tests/functional/src/main/java/org/mule/functional/junit4/runners/ArtifactClassLoaderRunnerConfig.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.functional.junit4.runners;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Specifies a configuration needed by {@link ArtifactClassloaderTestRunner} in order to
+ * run the test.
+ *
+ * @since 4.0
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@Inherited
+public @interface ArtifactClassLoaderRunnerConfig
+{
+
+    /**
+     * @return a comma separated list of packages to be added as PARENT_ONLY for the
+     * container class loader, default (and required) packages is empty.
+     * In case of having to append one you should also include the default list.
+     */
+    String extraBootPackages() default "";
```

Comment:
It is not here but it is in integration-tests branch where a few
extraBootPackages had to be added. The idea is to leave this option to
allow adding extraBootPackages in addition to the fixed ones defined in
ContainerClassLoader plus the one in excluded.properties file in
functional-tests.

On Wed, Jul 6, 2016 at 8:09 PM, Pablo Kraan <notifications@github.com>
wrote:

> In
> tests/functional/src/main/java/org/mule/functional/junit4/runners/ArtifactClassLoaderRunnerConfig.java
> <https://github.com/mulesoft/mule/pull/2930#discussion_r69827950>:
>
> > + * run the test.
> > + *
> > + * @since 4.0
> > + */
> > +@Retention(RetentionPolicy.RUNTIME)
> > +@Target(ElementType.TYPE)
> > +@Inherited
> > +public @interface ArtifactClassLoaderRunnerConfig
> > +{
> > +
> > +    /**
> > +     * @return a comma separated list of packages to be added as PARENT_ONLY for the
> > +     * container class loader, default (and required) packages is empty.
> > +     * In case of having to append one you should also include the default list.
> > +     */
> > +    String extraBootPackages() default "";
>
> Seems like this method is not used
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mulesoft/mule/pull/2930/files/36cf4aaec51b40210b3ba4499311604d8e9af264#r69827950>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG96KV8RDdngLAubVaqRXQ2EQeNqEk6Vks5qTDWogaJpZM4JGYaT>
> .
>



-- 
<http://www.mulesoft.com/>
Guillermo Fernandes, Senior Software Engineer

Av. Corrientes 316 Entre Piso, Buenos Aires, C1043AAQ, Argentina


<https://www.linkedin.com/company/www.mulesoft.com>
<https://twitter.com/mulesoft>   <https://www.facebook.com/MuleSoft>
<https://www.youtube.com/user/mulesoftvids> We're hiring!
<http://www.mulesoft.com/careers>